### PR TITLE
refactor(e2e Dashboard): fix Get Started button name

### PR DESCRIPTION
**Pull request by rukshan99**
### What does this PR do?
- Changes the button name from **Get Started Page** to **Get Started** in the left navigation panel in the Che Dashboard.

### Screenshot/screencast of this PR

Screenshot (275)

### What issues does this PR fix or reference?

Closes issue #19412 

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Comment by sleshchenko** [comment]
@rukshan99 Thanks for proving a fix!
Before we can move forward and merge your PR, you should fix Eclipse ECA. See https://accounts.eclipse.org/legal/eca/validation/63468

You can sign it on https://accounts.eclipse.org/user/eca and then you'll need to do
```
git commit --amend --sign-off
git push -f
```

**Comment by rukshan99** [comment]
> @rukshan99 Thanks for proving a fix!
> Before we can move forward and merge your PR, you should fix Eclipse ECA. See https://accounts.eclipse.org/legal/eca/validation/63468
> 
> You can sign it on https://accounts.eclipse.org/user/eca and then you'll need to do
> 
> ```
> git commit --amend --sign-off
> git push -f
> ```

@sleshchenko Thanks for the reply. I hope now it's okay.

File: tests/e2e/pageobjects/dashboard/Dashboard.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ import { Logger } from '../../utils/Logger';
 @injectable()
 export class Dashboard {
     private static readonly WORKSPACES_BUTTON_XPATH: string = `//div[@id='page-sidebar']//a[text()='Workspaces']`;
-    private static readonly GET_STARTED_BUTTON_XPATH: string = `//div[@id='page-sidebar']//a[text()='Get Started Page']`;
+    private static readonly GET_STARTED_BUTTON_XPATH: string = `//div[@id='page-sidebar']//a[text()='Get Started']`;
```
Comment (sleshchenko) [inline tests/e2e/pageobjects/dashboard/Dashboard.ts:23]: @rukshan99 as far as I understand you adapted e2e tests to changes on dashboard which are not done yet.

Would you mind to create another PR against che-dashboard repo,
Get Started Page appears here https://github.com/eclipse/che-dashboard/blob/master/src/Layout/Navigation/MainList.tsx#L27 and in unit tests

**Comment by sleshchenko** [comment]
Dashboard part is fixed in https://github.com/eclipse/che-dashboard/pull/202

File: tests/e2e/pageobjects/dashboard/Dashboard.ts (hunk 1 above)
Comment (sleshchenko) [reply tests/e2e/pageobjects/dashboard/Dashboard.ts:23]: Dashboard side is fixed by https://github.com/eclipse/che-dashboard/pull/202

**Review by sleshchenko — APPROVED** [review]

**Pull request merged**